src/app/code/community/Divante/VueStorefrontIndexer/Model/Index/Mapping/Eav/Abstract.php

```diff
@@ -51,18 +51,29 @@ public function getAttributeMapping(Attribute $attribute)
 
         $type = $this->getAttributeType($attribute);
 
-        if ($type === 'text' && !$attribute->getBackendModel() && $attribute->getFrontendInput() != 'media_image') {
+        if ($type === 'text' && !$attribute->getBackendModel() && $attribute->getFrontendInput() != 'media_image' && $attribute->getAttributeCode() != 'description') {
             $fieldName = $attributeCode;
             $mapping[$fieldName] = [
                 'type' => $type,
+                'fielddata' => true,
+                'analyzer' => "autocomplete",
+                'search_analyzer'=> "autocomplete_search",
                 'fields' => [
                     'keyword' => [
                         'type' => FieldInterface::TYPE_KEYWORD,
                         'ignore_above' => 256,
                     ]
                 ]
             ];
-        } else if ($type === 'date') {
+        } 
+        else if ($type === 'text' && $attribute->getAttributeCode() != 'image' && $attribute->getAttributeCode() != 'description') {
+            $fieldName = $attributeCode;
+            $mapping[$fieldName] = [
+                'type' => $type,
+                'fielddata' => true
+            ];
+        }
```

Comment:
fieldata should be set for almost all text fields by default? @pkarw what do you think?

Reply:
As per this code, fielddata is set to all text attributes. What would you make you think it doesnt @afirlejczyk ?

Reply:
Yes, I recall this same problem.
Most fields (used in filters) should have int/long types -> as we are exporting only option_ids (by default, more info in the previous comment.)
Did you have a problem here with select/multiselect attributes or with other attributes as well?